src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -130,6 +139,13 @@ public void canCloseAnOpenLoanByCheckingInTheItem() {
     assertThat("ID should be included for item",
       itemFromResponse.getString("id"), is(nod.getId()));
 
+    assertThat("New location should not equal old location",
+      homeLocation.getId().toString(), not(anotherLocationId));
+    assertThat("The item's temporary location ID should be updated",
+      nod.getJson().getString("temporaryLocationId"), is(anotherLocationId));
```

Comment:
Move this test below so that you can use `updatedNod`.

Reply:
Good tip, thank you.